rafthttp/transport.go

```diff
@@ -206,6 +210,34 @@ func (t *Transport) Stop() {
 	t.remotes = nil
 }
 
+// DropSend drops messages to the specified peer.
+func (t *Transport) DropSend(id types.ID) {
+	t.mu.RLock()
+	p, pok := t.peers[id]
+	if pok {
+		p.setPaused(true)
+	}
+	g, gok := t.remotes[id]
+	if gok {
+		g.paused = true
```

Comment:
Not now. I will try adding `Resume` method to `remote` to satisfy `Pausable` interface here https://github.com/coreos/etcd/blob/master/rafthttp/remote.go#L59-L61.

Thanks!